lib/rubocop/cop/style/trailing_comma.rb

```diff
@@ -71,10 +71,14 @@ def check_literal(node, kind)
         # rubocop:disable Metrics/MethodLength
         def check(node, items, kind, begin_pos, end_pos)
           sb = items.first.loc.expression.source_buffer
+
           after_last_item = Parser::Source::Range.new(sb, begin_pos, end_pos)
 
           return if heredoc?(after_last_item.source)
 
+          # Stop Execution if data structure constructed with brackets is empty
+          return if items.first.children.empty?
```

Comment:
My only concern is that, based on this implementation, `Foo.new({}, \n ... 5)` would pass this cop, and perhaps it should not?